presto-orc/src/main/java/com/facebook/presto/orc/metadata/HiveBloomFilter.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.metadata;
+
+import com.google.common.primitives.Longs;
+import org.apache.hadoop.hive.ql.io.orc.OrcProto;
+import org.apache.hive.common.util.BloomFilter;
+
+import java.util.List;
+
+public class HiveBloomFilter extends BloomFilter
+{
+    // constructor that allows unserializing a long list into the actual hive bloom filter
+    public HiveBloomFilter(List<Long> bits, int numBits, int numFuncs)
+    {
+        long[] copied = new long[bits.size()];
+        for (int i = 0; i < bits.size(); i++) {
+            copied[i] = bits.get(i);
+        }
+        this.bitSet = new BitSet(copied);
+        this.numBits = numBits;
+        this.numHashFunctions = numFuncs;
+    }
+
+    public HiveBloomFilter(BloomFilter bloomFilter)
+    {
+        this(Longs.asList(bloomFilter.getBitSet()), bloomFilter.getBitSize(), bloomFilter.getNumHashFunctions());
+    }
+
+    public OrcProto.BloomFilter toOrcBloomfilter()
```

Comment:
This appears to only be used in `TestOrcBloomFilters` so just move it there.  Something like:
```java
    private static OrcProto.BloomFilter toOrcBloomfilter(BloomFilter bloomFilter)
    {
        OrcProto.BloomFilter.Builder builder = OrcProto.BloomFilter.newBuilder();
        for (long bits : bloomFilter.getBitSet()) {
            builder.addBitset(bits);
        }
        builder.setNumHashFunctions(bloomFilter.getNumHashFunctions());
        return builder.build();
    }